.github/renovate.json

```diff
@@ -7,5 +7,11 @@
   "schedule": [
     "every weekend"
   ],
-  "groupName": "all"
+  "groupName": "all",
+  "packageRules": [
+    {
+      "matchPackageNames": ["com.github.ben-manes.caffeine:"],
+      "allowedVersions": "< 3.0.0"
```

Comment:
Shall we add a comment here for better understanding to remove this, when we upgrade our project to Java 11.

Reply:
Okay lets leave it as it is then. We can handle them later.